docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -562,7 +562,7 @@ kind: query
 spec:
   name: Detect active processes with Log4j running
   platforms: macOS, Linux
-  description: Returns a list of active processes and the Jar paths which are using Log4j. Version numbers are usually within the Jar filename.
+  description: Returns a list of active processes and the Jar paths which are using Log4j. Version numbers are usually within the Jar filename. Note: This query is resource intensive and has caused problems on systems with limited swap space. Test on some systems before running this widely.
```

Comment:
Ah, I think we broke things with the `:` character in here. I'll put up a PR to fix it.

```
fleetctl preview
Downloading dependencies from production into /Users/zwass/.fleet/preview...
Pulling Docker dependencies...
Starting Docker containers...
Waiting for server to start up...
Initializing server...
Configured fleetctl in the 'preview' context to avoid overwriting existing config.
Loading standard query library...
failed to parse standard query library: error converting YAML to JSON: yaml: line 6: mapping values are not allowed in this context
```